textproc/epubcheck/Portfile

```diff
@@ -1,43 +1,49 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem              1.0
+
 PortGroup               java 1.0
 
-name                    epubcheck
-version                 1.2
+PortGroup               github 1.0
+
+github.setup            w3c epubcheck 4.2.2 v
 categories              textproc
 platforms               darwin
-license                 BSD 
+license                 BSD-3-Clause
```

Comment:
I'm not sure about this `license` notation. No other port has it. I wonder if "BSD" is enough?

Reply:
According to [the wiki page](https://trac.macports.org/wiki/PortfileRecipes#licensekeyword), we only have "bsd" and "bsd-old". Are those different than a "3 clause" BSD license? In what way? As specified this port will not be distributable by Macports because it is not a known license type.

Reply:
this should just be ```BSD``` as it was originally

Reply:
`BSD-3-Clause` looks like an [SPDX](https://spdx.org) identifier. MacPorts doesn't use those yet; I don't think that existed when we decided on our license naming scheme. Transitioning to SPDX might be a good idea, but needs to be planned and coordinated properly. The values in the license field need to be ones that the [port_binary_distributable.tcl](https://github.com/macports/macports-infrastructure/blob/master/jobs/port_binary_distributable.tcl) included file [distributable_lib.tcl](https://github.com/macports/macports-infrastructure/blob/master/jobs/distributable_lib.tcl) recognizes, otherwise we won't be distributing archives we should be able to distribute. So please keep it at `BSD` until further notice.

Reply:
In MacPorts, `BSD` means the modern 2- or 3-clause BSD license, while `BSD-old` means the much older and now almost extinct 4-clause BSD license.